src/TraceEvent/Parsers/GCDynamicTraceEventParser.cs

```diff
@@ -340,7 +278,7 @@ internal GCDynamicEvent Bind(GCDynamicTraceEvent underlyingEvent)
         }
     }
 
-    internal sealed class RawDynamicTraceData : GCDynamicEvent
+    public sealed class RawDynamicTraceEvent : GCDynamicEvent
```

Comment:
If you are going to make this public, and we name it GCDynamicTraceEvent to match the event name, and scope it appropriately?